opengovsg/plumber
# PLU-89: Editor layout changes

**Things changed**:
- Change `Publish` button
- Change the left icon for the editor layout bar
- Change divider to use the border colour: `base.divider.subtle`
- Change border for EditableTypography to `secondary.100`
- Migrate some MUI components

## Before & After Screenshots

**BEFORE**:
<img width="1497" alt="image" src="https://github.com/opengovsg/plumber/assets/65110268/567d3282-e7c2-4106-9c0c-58c7f5919354">

**AFTER**:
<img width="1503" alt="image" src="https://github.com/opengovsg/plumber/assets/65110268/e7d403f0-aa1f-43f7-9fae-ba3cd0ad5960">

## Tests
Regression tests:
- Check whether pipe name can be changed
- Check whether pipe can be published
- Check whether pipe workflow can be activated
